src/nvim/os/fs.c

```diff
@@ -351,15 +367,6 @@ static bool is_executable_in_path(const char_u *name, char_u **abspath)
 #endif
 
   size_t buf_len = STRLEN(name) + strlen(path) + 2;
-
-#ifdef WIN32
-  const char *pathext = os_getenv("PATHEXT");
-  if (!pathext) {
-    pathext = ".com;.exe;.bat;.cmd";
-  }
-  buf_len += strlen(pathext);
```

Comment:
Ok to remove this because `is_executable_ext` works with `os_buf` so  `buf` doesn't need extra space.